dstopic/commands/cmd-hello/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+use dstopic_command::Command;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct HelloCmd {
+    arg: Option<String>,
+}
+
+impl Command for HelloCmd {
+    fn execute(self) -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
```

Comment:
IIRC from yoshua's error post, wasn't there a macro that hid the sort-of-intimidating-to-Rust-newcomers `Box<dyn std::error::Error + Send + Sync>` incantation?

Reply:
I think I'll just say fuck it and pull in `anyhow` for errors here. It makes things so much nicer and I've been having good experiences with it. I agree that this is quite an incantation, but I wasn't sure I wanted to commit to an error handling strategy just yet.